src/containers/PageContainer.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import React from 'react'
+import { ThemeProvider } from 'styled-components'
+import theme from '../styles/theme'
+// Base styles
+import '../styles/reset.css'
+import '../styles/style.css'
```

Comment:
Why do you need to import them here again?

Reply:
Nope, `_document` is only used in case of server side rendering.
I will move these imports in a `_app` file and remove them from here.